types/v56/constants.go

```diff
@@ -219,7 +219,7 @@ const (
 )
 
 const (
-	// The QT* constants are the names used with Query requests to retrieve the corresponding entities
+	// The Qt* constants are the names used with Query requests to retrieve the corresponding entities
```

Comment:
Maybe like this to note what "Qt" abbreviates:
```suggestion
	// The Qt* (query type) constants are the names used with Query requests to retrieve the corresponding entities
```

Reply:
added